specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -12241,10 +12272,37 @@
           "format": "date-time",
           "description": "The UTC timestamp when the operation began."
         },
-        "startedBy": {
```

Comment:
Just checking if this removal is deliberate and it's because we're waiting for confirmation from IDC on agreed upon values to Unknown, User and Platform,

Reply:
Not aware if these discussions, I merely went by the fact that it is no longer in our API response: https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=%2Fsrc%2FCRP%2FDev%2FComputeResourceProvider%2FApiEntityModel%2FVMGuestPatch%2FVMGuestPatchApiEntityModel.cs&version=GBmaster&line=122&lineEnd=122&lineStartColumn=18&lineEndColumn=54&lineStyle=plain&_a=contents